src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -5,28 +5,85 @@
 import * as React from 'react';
 import flow from 'lodash/flow';
 import { FormattedMessage } from 'react-intl';
+
+import { Text } from '@box/blueprint-web';
+import { AddMetadataTemplateDropdown } from '@box/metadata-editor';
+
+import API from '../../api';
+import { type ElementsXhrError } from '../../common/types/api';
+import { type ElementOrigin } from '../common/flowTypes';
 import { withAPIContext } from '../common/api-context';
 import { withErrorBoundary } from '../common/error-boundary';
 import { withLogger } from '../common/logger';
 import { ORIGIN_METADATA_SIDEBAR_REDESIGN } from '../../constants';
 import { EVENT_JS_READY } from '../common/logger/constants';
 import { mark } from '../../utils/performance';
+import LoadingIndicator from '../../components/loading-indicator/LoadingIndicator';
+import InlineError from '../../components/inline-error/InlineError';
```

Comment:
we should be using blueprint components instead of the buie components

Reply:
both components changed